src/revivalpmmp/pureentities/data/BiomeInfo.php

```diff
@@ -73,4 +73,4 @@ class BiomeInfo{
 		Data::NETWORK_IDS["spider"],
 		Data::NETWORK_IDS["zombie"]
 	];
-}
\ No newline at end of file
+}
```

Comment:
no ending lines

Reply:
I'm editing online in the code editor, for the life of me I can't remove these (nor did I add them)?

Reply:
Confirmed, no empty new line at end of file?